x-pack/plugins/cloud_security_posture/server/routes/status/status.ts

```diff
@@ -105,31 +127,70 @@ const getCspStatus = async ({
     findingsLatestIndexStatus,
     findingsIndexStatus,
     scoreIndexStatus,
+    findingsLatestIndexStatusCspm,
+    findingsIndexStatusCspm,
+    scoreIndexStatusCspm,
+    findingsLatestIndexStatusKspm,
+    findingsIndexStatusKspm,
+    scoreIndexStatusKspm,
     installation,
     latestCspPackage,
-    installedPackagePolicies,
+    installedPackagePoliciesKspm,
+    installedPackagePoliciesCspm,
     installedPolicyTemplates,
   ] = await Promise.all([
     checkIndexStatus(esClient.asCurrentUser, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger),
     checkIndexStatus(esClient.asCurrentUser, FINDINGS_INDEX_PATTERN, logger),
     checkIndexStatus(esClient.asCurrentUser, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger),
+
```

Comment:
Do  we need the
  `checkIndexStatus(esClient.asCurrentUser, LATEST_FINDINGS_INDEX_DEFAULT_NS, logger),
    checkIndexStatus(esClient.asCurrentUser, FINDINGS_INDEX_PATTERN, logger),
    checkIndexStatus(esClient.asCurrentUser, BENCHMARK_SCORE_INDEX_DEFAULT_NS, logger),` if we are checking for the 'kspm' and 'cspm' statuses below?

Reply:
those 3 are for the indicesDetails field on the response (which is not under cspm or kspm field)